### PR TITLE
Add bug report guidelines

### PR DESCRIPTION
**Pull request by peyerluk**

### CODE REVIEW

File: collaboration/bug-reports.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+
+## How to describe a bug
+
+
+
+- Describe the actions you performed (Steps)  
+  Add Animated GIFs and Images if possile.  
+  Add more information if the bug is hard to reproduce.
+- Describe the Expected and Actual Results
+- Don't fix bugs in feature branches (and if you do create a bug ticket first)
```
Comment (masone) [inline collaboration/bug-reports.md]: I don't get this, maybe it needs clarificatino. A PR with a fix should be favored over a bug ticket, right?

File: collaboration/bug-reports.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+
+## How to describe a bug
```
Comment (marcbachmann) [inline collaboration/bug-reports.md]: I'd also define how to write the title of a bug ticket

File: collaboration/bug-reports.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+If necessary add a description describing the state of the app and what you were trying to do.
+
+If you have a static issue add images to provide context to your bug. Highlight the areas of interest in your image.
+
+
+## How to proceed when you fix a bug while working on a feature branch?
+
+You basically have two options:
+
+1. Go back to the master branch and create a new pr for the bugfix. Create the description of your pull request as if it where a bug issue. If you need the bugfix in your branch, cherry pick the fix (or merge the whole bugfix-branch into your feature branch).
```
Comment (marcbachmann) [inline collaboration/bug-reports.md]: I prefer to merge it. That way you don't duplicate the commits.
So I'd move the cherry-pick into the Parentheses.

File: collaboration/bug-reports.md (hunk 1 above)
Comment (peyerluk) [inline collaboration/bug-reports.md]: I'll try to rephrase. Goal is that we don't have bugfixes lying in feature branches while others don't know about it and maybe fix the same bug as well.

**Pull request merged**